.github/workflows/build.yml

```diff
@@ -14,6 +14,34 @@ jobs:
           repository: moergo-sc/zmk
           ref: main
           path: src
+          fetch-depth: 0
```

Comment:
Adding `fetch-depth: 0` here means that the history of the repository will be retrieved, which is necessary to resolve three-way merges.